doc/flame/examples/lib/main.dart

```diff
@@ -50,6 +52,8 @@ void main() {
     'router': RouterGame.new,
     'scale_by_effect': ScaleByEffectGame.new,
     'scale_to_effect': ScaleToEffectGame.new,
+    'size_by_effect': SizeByEffectGame.new,
+    'size_to_effect': SizeToEffectGame.new,
```

Comment:
These shouldn't be indented

Reply:
Not sure what you mean here, don't they follow the other routes' pattern?

Reply:
It looked super weird when reviewing it on the phone, looks completely fine in reality!